assets/src/components/v2/pre_fare/simulation_screen_container.tsx

```diff
@@ -45,7 +45,7 @@ const SimulationScreenLayout: ComponentType<SimulationScreenLayoutProps> = ({
           <div className="simulation__full-page">
             <div className="simulation__title">Live view</div>
             <div
-              className={classWithModifier("simulation", getScreenSide())}
+              className={`simulation + ${classWithModifier("simulation-viewport", getScreenSide())}`}
```

Comment:
This is mixing string concatenation and interpolation; the result will be

```
<div class="simulation + simulation-viewport simulation-viewport--duo" ...>
```

In addition to the intended classes, the element has a class of `+` (technically valid, but not useful). I think you meant one of these two:

```
className={`simulation ${classWithModifier("simulation-viewport", getScreenSide())}`}
```

```
className={"simulation " + classWithModifier("simulation-viewport", getScreenSide())}
```